freebasics/templates/base.html

```diff
@@ -1,124 +1,122 @@
 
 {% load wagtailcore_tags compress static wagtailuserbar core_tags freebasics_tags i18n %}
+{% load static from staticfiles %}
 
 <!DOCTYPE html>
 <!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-    <head>
-        <meta charset="utf-8" />
-        <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}{% block title_suffix %}{% endblock %}</title>
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="description" content="" />
-        <meta name="keywords" content="" />
-        <meta property="og:title" content="" />
-        <meta property="og:description" content="" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-        {% compress css %}
-            <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
-        {% endcompress %}
-
-        {% block extra_css %}
-        {% endblock %}
-        {% custom_css %}
```

Comment:
You still need this - also, please test that the colors set using the env variables work.
You can do something like `CUSTOM_CSS_ACCENT_1=#999 ./manage.py runserver`